package.json

```diff
@@ -26,6 +26,7 @@
     "webpack": "5"
   },
   "devDependencies": {
-    "husky": "^9.0.11"
+    "husky": "^9.0.11",
+    "util": "^0.12.5"
```

Comment:
> [!note]
> This was added to address the [react-components node.js error](https://github.com/department-of-veterans-affairs/component-library/actions/runs/15645401046/job/44081850440?pr=1642): 
> > ERROR in ../../node_modules/sinon/lib/sinon/util/core/format.js 3:14-37
Module not found: Error: Can't resolve 'util' in '/home/runner/work/component-library/component-library/node_modules/sinon/lib/sinon/util/core'